packages/kununu-utils/middlewares/expressLogger/index.js

```diff
@@ -21,7 +21,9 @@ const expressLogger = label => (req, res, next) => {
     res.removeListener('close', log);
     res.removeListener('error', log);
 
-    logger.log(this.status, {req, res, label, timeTakenMicros: (new Date() - startDate) * 1000});
+    const status = res.statusCode < 400 ? 'info' : 'error';
```

Comment:
This is necessary to correctly categorize errors, otherwise it will all be logged as `info`